pasqal_cloud/client.py

```diff
@@ -36,13 +36,22 @@ class Client:
 
     def __init__(
         self,
-        group_id: str,
         username: Optional[str] = None,
         password: Optional[str] = None,
         token_provider: Optional[TokenProvider] = None,
         endpoints: Optional[Endpoints] = None,
         auth0: Optional[Auth0Conf] = None,
+        group_id: Optional[str] = None,
+        project_id: Optional[str] = None
     ):
+        # Ticket (#622), used to avoid a breaking change during the group to project renaming
+        if not (project_id or group_id):
+            raise TypeError("Either a group_id or project_id has to be given as argument")
+
+        # Ticket (#622),to be removed as well
+        if not project_id:
+            project_id = group_id
```

Comment:
The client should be called directly by users of the sdk. Therefore I think it would be ok to only allow `project_id` here and remove these checks/conversion